genome_designer/genome_finish/graph_contig_placement.py

```diff
@@ -55,6 +102,37 @@ def graph_contig_placement(contig_list, skip_extracted_read_alignment,
         subprocess.check_call(' '.join(['cat'] + contig_fastas),
                 shell=True, executable=settings.BASH_PATH, stdout=fh)
 
+    # Write mobile elements to file
+    ref_genome = sample_alignment.alignment_group.reference_genome
+    genbank_query = ref_genome.dataset_set.filter(
+            type=Dataset.TYPE.REFERENCE_GENOME_GENBANK)
+
```

Comment:
Please add a comment to say something about mobile elements, in case it's not obvious what 'me' means.